apps/network-alerts/test/unit/notification/notification_test.js

```diff
@@ -34,7 +34,8 @@ suite('Network Alerts - Notification handling', function() {
     var message = {
       title: 'Some title',
       body: 'Some body',
-      clicked: true
+      clicked: true,
```

Comment:
__nit:__ you don't need the piece of code above now since you don't parse URL anymore (hence the change in the test will be needed, will leave it to you to figure out what exactly :))
```
this.sinon.stub(Utils, 'parseParams').returns({
       titleID: 'titleID'
     });
```